shared/templates/csv/auditd_lineinfile.csv

```diff
@@ -0,0 +1,2 @@
+# <rule_id, parameter, value[, missing_parameter_pass]> (missing_parameter_pass: If set, the OVAL check will also pass if the parameter is not present in the configuration file. Values can be true or false)
```

Comment:
Can you please add, that false is default value? (also for sshd description)

Reply:
Added for both `csv` files.